### ISSUE TITLE
功能建议：支持超宽显示

### ISSUE THREAD

**Issue by weey007** [chathub-dev/chathub#108]
我的屏幕是宽屏，目前只能在屏幕中间展示。能否支持 超宽显示 选项

**Comment by wong2** [comment]
可以截图现在的显示效果看看吗

**Comment by meesii** [comment]
+1，我34寸屏幕看着很难受。

**Comment by wong2** [comment]
可以截图或拍照看看吗

**Comment by meesii** [comment]
看了css有一个max-width: 1316px限制了

**Comment by wong2** [comment]
对。像您这种情况，如果真的把屏幕宽度全占了，也会有点过于宽了吧？

**Comment by meesii** [comment]
不需要全占，我正常情况屏幕是1:1分屏。但是一半屏幕的时候也占不满。对我来说就感觉很小了。
以下截图是我一般屏幕的大小，两边空了太多了。
![image](https://user-images.githubusercontent.com/1815046/226496414-8b1ec0ef-73e0-48e7-8e8f-89e8ef9e1f2b.png)

**Comment by EazyCheeze1978** [comment]
Yes, this will especially come in handy when you begin to implement more than two chat bots on the all-in-one screen. Note here, I've engaged the Nvidia surround mode so that I can have a simulated ultra wide experience with my multiple monitors, and that the all-in-one window is centered and quite large; when the ability for the third or fourth chat bot in all-in-one is added, it might help to have a more even display for that; even with only two, if
![20230420_055422](https://user-images.githubusercontent.com/37255420/233347054-f15981f0-9690-49f5-b6f1-b8997b9941ab.jpg)
 the CSS or whatever has the ability to correctly split this window such that the left chat bot appears only on the left monitor, and the right on the right, and the text entry field for sending to all bots appears on the user's choice of left or right monitor, that would also be really cool.

Oh, and please don't mind the state of my desk :-)

![20230420_055422](https://user-images.githubusercontent.com/37255420/233347054-f15981f0-9690-49f5-b6f1-b8997b9941ab.jpg)
